src/layouts/ModelizerDrawLayout.vue

```diff
@@ -89,6 +104,16 @@ async function initView() {
   ]);
 }
 
+/**
+ * Rearrange components on click,
+ * and then redraw the whole view.
+ * @param {string} targetId - Id of the drawing target.
+ */
+async function arrangeComponentsPosition(targetId) {
```

Comment:
parameter not necessary (cf. `ModelizerDrawPage.vue`)

Reply:
removed